src/main/java/com/billingapi/controller/InvoiceController.java

```diff
@@ -1,115 +1,65 @@
 package com.billingapi.controller;
 
-import java.util.List;
-
+import com.billingapi.controller.request.InvoiceRequest;
+import com.billingapi.controller.request.InvoiceResponse;
+import com.billingapi.exception.InvoiceNotFoundException;
+import com.billingapi.model.Invoice;
+import com.billingapi.service.InvoiceService;
+import com.billingapi.service.InvoiceServiceImpl;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.validation.annotation.Validated;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.PutMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
-import com.billingapi.controller.request.InvoiceRequestDto;
-import com.billingapi.controller.request.InvoiceResponseDto;
-import com.billingapi.model.Invoice;
-import com.billingapi.service.InvoiceService;
-import com.billingapi.service.InvoiceServiceImpl;
+import javax.validation.Valid;
 
 @RestController
-@EnableAutoConfiguration
 @RequestMapping("/invoice")
 @Validated
 public class InvoiceController {
-	@Autowired
-	private InvoiceServiceImpl invoiceService;
+    @Autowired
+    private InvoiceService invoiceService;
+
 
-	@GetMapping("/{invoiceId}")
-	public ResponseEntity<?> findInvoicetById(@PathVariable int invoiceId) {
-		Invoice invoice = invoiceService.findInvoicetById(invoiceId);
-		return new ResponseEntity<>(invoiceService.findInvoicetById(invoiceId), HttpStatus.OK);
-	}
+    //To find Invoice
```

Comment:
Can you write some meaningful method comment here .. Please have a look https://www.oracle.com/in/technical-resources/articles/java/javadoc-tool.html

Reply:
Apply to other places also